framework/framework.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	CRDClient *k8scrdclient.CRDClient
 	Informer  informer.Interface
 	Logger    micrologger.Logger
+	Name      string
```

Comment:
This refers to the operator name, we now add this to the finalizer to differentiate between operators.

Reply:
Would be cool to add a setting to the settings block. This here is for dependencies. The difference is one implements deeper business logic, the other is just about data.

Reply:
Godoc please. Also explaining why it needs to be unique <3

Reply:
I added some godoc ( not sure if it's clear or enough) and moved it down.